addons/l10n_latam_check/tests/test_third_party_checks.py

```diff
@@ -0,0 +1,177 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo.addons.l10n_latam_check.tests.common import L10nLatamCheckTest
+from odoo.exceptions import ValidationError, UserError
+from odoo.tests.common import tagged
+
+
+@tagged('post_install_l10n', 'post_install', '-at_install')
+class TestThirdChecks(L10nLatamCheckTest):
+
+    def create_third_party_check(self, journal=False):
+        if not journal:
+            journal = self.third_party_check_journal
+        vals = {
+            'partner_id': self.partner_a.id,
+            'amount': '00000001',
+            'check_number': '00000001',
+            'payment_type': 'inbound',
+            'journal_id': journal.id,
+            'payment_method_line_id': self.third_party_check_journal._get_available_payment_method_lines('inbound').filtered(lambda x: x.code == 'new_third_party_checks').id,
+        }
+        payment = self.env['account.payment'].create(vals)
+        payment.action_post()
+        return payment
+
+    def test_01_get_paid_with_multiple_checks(self):
+        vals_list = [{
+            'partner_id': self.partner_a.id,
+            'amount': '00000001',
+            'check_number': '00000001',
+            'payment_type': 'inbound',
+            'journal_id': self.third_party_check_journal.id,
+            'payment_method_line_id': self.third_party_check_journal._get_available_payment_method_lines('inbound').filtered(lambda x: x.code == 'new_third_party_checks').id,
+        }, {
+            'partner_id': self.partner_a.id,
+            'amount': '00000002',
+            'check_number': '00000002',
+            'payment_type': 'inbound',
+            'journal_id': self.third_party_check_journal.id,
+            'payment_method_line_id': self.third_party_check_journal._get_available_payment_method_lines('inbound').filtered(lambda x: x.code == 'new_third_party_checks').id,
+        }]
+        payments = self.env['account.payment'].create(vals_list)
```

Comment:
we can't use it because that method creates a generic check one by one, what we are testing here is creating two checks at the same time to test the create multi